emscripten/luna.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en-us">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <title>luna-web</title>
+  <script src="FileSaver.js" type="text/javascript"></script>
+</head>
+
+<body>
+
+<div>
+    Create a .tns file from a .lua script: <input type="file" id="luaFile" accept=".lua" onChange="fileLoad(event)"/><br>
```

Comment:
This works just fine for Problem1.xml without any modifications, so maybe accept both?